static/treemap.js

```diff
@@ -22,6 +22,12 @@ var svg = d3.select("#body").append("div")
         .append("svg:g")
         .attr("transform", "translate(.5,.5)");
 
+var valuefns = {
+    "cpu": (d => d.cpu),
+    "soft-mem": (d => d.soft_memory || d.max_memory),
+    "max-mem": (d => d.name == "Unused" ? 0 : d.max_memory) // Hide Unused node
```

Comment:
why hide? might it be useful to see unused nodes so we could remove them?

Reply:
When displaying "max-mem" values, the notion of "unused" space is confusing I felt.  Often a node is over allocated so showing a gap was misleading, imo.  To find nodes that are completely empty it made more sense to look at "soft-mem" or "cpu" allocations since neither of those values can go over 100%.